models/room.js

```diff
@@ -31,11 +32,11 @@ const Room = mongoose.model('Room', roomSchema)
 
 const validateRoom = (room) => {
   const schema = Joi.object({
-    hotelId: Joi.objectId().required(),
-    beds: Joi.object({
+    hotelId: Joi.objectId(),
```

Comment:
actually it should be required. We dont want to have situation that room will be without hotel